src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Truck;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer {

```

Comment:
The `TruckProducer` class should parameterize the `MachineProducer` interface with the `Truck` type. According to the task requirements, you should replace `Object` in `get()` with the suitable option, such as `List<Truck> get();`.